src/commandParser.py

```diff
@@ -152,7 +152,7 @@ def __argCheck(self, args, inoutInfo, subservice=None):
 
     def __lexer(self, input):
         tokenList = []
-        tmp = re.split(r'([a-zA-Z_-]+|[\(\)]|[0-9_.-]*)', input)
+        tmp = re.split('([-.]+|[0-9.]+|[a-zA-Z0-9_-]+|[\(\)])', input)
```

Comment:
1. `autopep8` inserts `r` at the beginning which messed up the whole thing. So, I removed it.
2. The first bracket is for morse message. The second for float numbers. The third for alphanumerical arguments. The order must be as this so the command parser works correctly. However, please check and let me know if it ruins some other part of the code.

Reply:
As long as you've tested it and the commands still work as expected, it should be fine